### DISCUSSION TITLE
Coin holder benefit when NFT Sells

### DISCUSSION THREAD

**Discussion by ImJigarShah** [deso-protocol/dips#124] [Medium CIPs]
Hey, I have a suggestion from so many days I have bought NFT artist coins eg. diamond hands but what happens when diamond hands sell his NFT so his coin holder get benefits 10% of the total amount selling so there is a lack of knowledge where this 10% is going because I didn't yet get my coin holder benefits from any of such artist who has sold his/her NFT only got gain when someone buys & loss when his/her coin sell.

Suggestion-
1) Give notification x amount of coin added to your coins your holding.
2) Give clouts instead of coins (but some sort of notification is needed so the investor can check he is getting a benefit or not)

**Comment by ItsAditya-xyz** [comment]
My take :

- At present, it's hard to notice what you have gained as a coin holder of an NFT artist. It is not that big amount that you can notice as of now. Let say you hold 20% of the coins of an NFT artists who sells his NFT for an average of $100/piece with 10% coin holder royalty. On every sale you as a coin holder get $2 in the NFT artist's creator coin. This gain is hard to notice because other investors may buy/sell his coin, fluctuating CLOUT price etc. 

- Yes, it would be cool to get notification for the amount one gets as a coin holder of an NFT artist as this will bring more transparency. 

- Coin holder royalty in CLOUT? Oh well, I think this requires more community discussion

**Reply by ImJigarShah** [reply]
Agree with your points need suggestion on core devs team this will make more transparency in bitclout.

**Reply by danwils33** [reply]
I think they avoided making coin-holder royalties being paid out in $clout because that would make creator coins a security and they don't want to mess with that.

**Reply by AikonMishima** [reply]
1. Notifications upon sale would make sense.
2. Royalty payment in $Clout will reduce the use of a creator coin, and as per @danwils33 ’ reply, could make it a security.

When attracting the masses we want more functionalities for the creator coin, not less.

**Reply by ImJigarShah** [reply]
Yes, Agree but need notification x amount of creator coin added as your holder of X person who sold NFT.

**Reply by arhebbar** [reply]
As far as I know, no coin is added to anyone. If 100$ of Royalty needs to go to a Creator Coin's Holders, the amount of coins are bought and then burnt so that the base price goes up for everyone. So, as of now, a notification on NFT Selling going to a Creator and a NFT selling and the amount of Royalty on it going to Coin Holders should be sufficient.
